ncclient/operations/rpc.py

```diff
@@ -205,6 +205,10 @@ def register(self, id, rpc):
 
     def callback(self, root, raw):
         tag, attrs = root
+        ind = tag.find("}")
```

Comment:
Why not just:

```python
if tag.find('rpc-reply') != -1:
    return
```

As an aside, I suspect that all the device handlers that are doing:

```python
def perform_qualify_check(self):
    return False
```

...have misunderstood what this function is for. This is really a way for a device to say "I don't return valid XML, so please skip namespace-aware validating checks". Certainly for the device types we're looking at, `perform_qualify_check` should return `True`.

Maybe we should fix the device handlers as well? At least the Cisco ones?

Reply:
Actually, just pulled latest changes from `ncclient/ncclient`, and device handlers have been fixed. I think I got one of the guys working on YANG Suite to do that a few months ago!! 👍